adal/proguard-rules.pro

```diff
@@ -1,11 +1,8 @@
-# To enable ProGuard in your project, edit project.properties
-# to define the proguard.config property as described in that file.
-#
 # Add project specific ProGuard rules here.
```

Comment:
Do we have any proguard rules here? I wonder if we don't have rules, do we actually need it?